frontend/src/js/components/tenants/TenantCreateForm.tsx

```diff
@@ -0,0 +1,208 @@
+// Copyright 2024 Northern.tech AS
+//
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+//
+//        http://www.apache.org/licenses/LICENSE-2.0
+//
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+import React, { Dispatch, SetStateAction, useEffect, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { Link } from 'react-router-dom';
+
+import { Close as CloseIcon } from '@mui/icons-material';
+import { Divider, Drawer, IconButton, TextField } from '@mui/material';
+import { makeStyles } from 'tss-react/mui';
+
+import Api from '@northern.tech/store/api/general-api';
+import { getOrganization } from '@northern.tech/store/organizationSlice/selectors';
+import { addTenant } from '@northern.tech/store/organizationSlice/thunks';
+import { AppDispatch } from '@northern.tech/store/store';
+import { useradmApiUrlv1 } from '@northern.tech/store/usersSlice/constants';
```

Comment:
```suggestion
import { getOrganization } from '@northern.tech/store/selectors';
import { addTenant } from '@northern.tech/store/thunks';
import { useradmApiUrlv1 } from '@northern.tech/store/constants';
import { AppDispatch } from '@northern.tech/store/store';
```

as the internal package structure might change, the imports here should target the outer exports 👌 - maybe you can configure your editor to prefer these?